tests/v2/param_mockgen.js

```diff
@@ -13,7 +13,7 @@ describe('Parameter Mock generator', () => {
             path: '/store/order/{orderId}',
             operation: 'get'
         }, (err, mock) => {
-            Assert.ok(!err, 'No error');
```

Comment:
This will help you actually see the errors that are coming in, making debugging a lot easier. 

Also [should.js](https://shouldjs.github.io/) is a fantastic assertions library. Check out [oas-kit](https://github.com/Mermade/oas-kit) and [Speccy](https://github.com/wework/speccy) for loads of actual usage examples with OAS world.